include/lsst/afw/image/PhotoCalib.h

```diff
@@ -95,45 +95,45 @@ class PhotoCalib : public table::io::PersistableFacade<PhotoCalib>, public table
      *
-     * @param[in]  instFluxMag0     The constant instFlux/magnitude zero point (instFlux at magnitude 0).
-     * @param[in]  instFluxMag0Err  The error on the zero point.
+     * @param[in]  calibrationMean  The spatially-constant calibration (equal to 1/(instFlux at magnitude 0)).
+     * @param[in]  calibrationError  The error on the zero point.
      * @param[in]  bbox             The bounding box on which this PhotoCalib is valid. If not specified,
      *                              this PhotoCalib is valid at any point (i.e. an empty bbox).
      */
-    explicit PhotoCalib(double instFluxMag0, double instFluxMag0Err = 0,
+    explicit PhotoCalib(double calibrationMean, double calibrationError = 0,
                         afw::geom::Box2I const &bbox = afw::geom::Box2I())
-            : _instFluxMag0(instFluxMag0), _instFluxMag0Err(instFluxMag0Err), _isConstant(true) {
+            : _calibrationMean(calibrationMean), _calibrationErr(calibrationError), _isConstant(true) {
         ndarray::Array<double, 2, 2> coeffs = ndarray::allocate(ndarray::makeVector(1, 1));
-        coeffs[0][0] = instFluxMag0;
-        _zeroPoint = std::make_shared<afw::math::ChebyshevBoundedField>(
+        coeffs[0][0] = calibrationMean;
+        _calibration = std::make_shared<afw::math::ChebyshevBoundedField>(
                 afw::math::ChebyshevBoundedField(bbox, coeffs));
     }
 
     /**
      * Create a spatially-varying calibration.
      *
-     * @param[in]  zeroPoint       The spatially varying photometric zero point.
-     * @param[in]  instFluxMag0Err The error on the zero point.
+     * @param[in]  calibration      The spatially varying photometric zero point.
+     * @param[in]  calibrationError The error on the zero point.
      */
-    PhotoCalib(std::shared_ptr<afw::math::BoundedField> zeroPoint, double instFluxMag0Err = 0)
-            : _zeroPoint(zeroPoint),
-              _instFluxMag0(computeInstFluxMag0(zeroPoint)),
-              _instFluxMag0Err(instFluxMag0Err),
+    PhotoCalib(std::shared_ptr<afw::math::BoundedField> calibration, double calibrationError = 0)
+            : _calibration(calibration),
+              _calibrationMean(computeCalibrationMean(calibration)),
+              _calibrationErr(calibrationError),
               _isConstant(false) {}
 
     /**
      * Create a calibration with a pre-computed mean. Primarily for de-persistence.
      *
-     * @param[in]  instFluxMag0    The mean of the zeroPoint() over its bounding box.
-     * @param[in]  zeroPoint       The spatially varying photometric zero point.
-     * @param[in]  instFluxMag0Err The error on the zero point.
-     * @param[in]  isConstant      Is this PhotoCalib spatially constant?
+     * @param[in]  calibrationMean  The mean of the calibration() over its bounding box.
+     * @param[in]  calibration      The spatially varying photometric zero point.
+     * @param[in]  calibrationError The error on the zero point.
```

Comment:
Thanks for catching these: I've removed all the erroneous references to "zero point", because that's not a useful term here any more (except the one method that returns `instFluxMag0`). I'm leaving the mathematical description for the class docstring at the top, so it only has to be written once.